[core]/es_extended/client/functions.lua

```diff
@@ -140,12 +140,12 @@ function ESX.RefreshContext(...)
 end
 
 function ESX.RegisterInput(command_name, label, input_group, key, on_press, on_release)
-    RegisterCommand("+" .. command_name, on_press)
-    Core.Input[command_name] = ESX.HashString("+" .. command_name)
+    RegisterCommand(on_release and '+' .. command_name or command_name, on_press)
+    Core.Input[command_name] = ESX.HashString(command_name)
     if on_release then
-        RegisterCommand("-" .. command_name, on_release)
+        RegisterCommand('-' .. command_name, on_release)
     end
-    RegisterKeyMapping("+" .. command_name, label or '', input_group or 'keyboard', key or '')
+    RegisterKeyMapping(command_name, label or '', input_group or 'keyboard', key or '')
```

Comment:
Why removed '+' if you used 143 row?

Reply:
Made some testing, and apparently the plus sign is not required for **RegisterKeyMapping**. Tested in both cases where on_release is present and not.
Identical case for **HashString** inside of **RegisterInput**.
The only case where FiveM tries to trigger **on_release** is when the command name in **RegisterCommand** starts with a plus sign (+).
When **on_release** is not present, the command name in **RegisterCommand** doesn't use the symbol to prevent the following bug
![image](https://github.com/user-attachments/assets/2d289d9d-5ccb-4cda-a1c7-da377cc8b111)

Reply:
Thanks for test, and research, i merged.